Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/submission-box/SubmissionBox.tsx

```diff
@@ -196,6 +197,16 @@ const SubmissionBox = () => {
                 return null;
             }
 
+            if (invalidExtensionError) {
+                return (
```

Comment:
You could make the rendering of the alert box a separate function and call it everywhere it is needed to avoid repetition of code.